pkg/metrics/shoot.go

```diff
@@ -353,3 +361,12 @@ func (c gardenMetricsCollector) getSeeds() map[string]*gardenv1beta1.Seed {
 	}
 	return seeds
 }
+
+func shootIsCompliant(constraints *[]gardenv1beta1.Condition) string {
```

Comment:
```suggestion
func shootIsCompliant(constraints []gardenv1beta1.Condition) string {
```
Nit: I think there is no benefit of using a pointer to an array.

Reply:
good point